Description: Create the file test.c in the upstream tree
 State as of trickle 1.07-10.1

--- trickle-1.07.orig/test.c
+++ trickle-1.07/test.c
@@ -0,0 +1,20 @@
+#include <sys/types.h>
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif /* HAVE_CONFIG_H */
+
+#include <sys/tree.h>
+#include <sys/queue.h>
+#include <sys/socket.h>
+#include <sys/un.h>
+#include <sys/uio.h>
+#include <sys/stat.h>
+#include <sys/param.h>
+#ifdef HAVE_SYS_TIME_H
+#include <sys/time.h>
+#endif /* HAVE_SYS_TIME_H */
+
+#include <netinet/in.h>
+#include <netinet/in_systm.h>
+
